lib/types.ts

```diff
@@ -18,6 +18,23 @@ export type Profile = {
   statusMessage: string;
 };
 
+/**
+ * Request body which is sent by webhook.
+ *
+ * @see [Request body](https://developers.line.me/en/reference/messaging-api/#request-body)
+ */
+export type WebhookRequestBody = {
+  /**
+   * User ID of a bot that should receive webhook events. The user ID value is a string that matches the regular expression, U[0-9a-f]{32}.
+   */
+  destination?: string;
```

Comment:
Was destination optional?

Reply:
Actually, not. I thought it may not be provided in old versions of LINE, but it may be independent from LINE client version. I'll fix change it to be required.